Entities/Natural/Seed/Seed.as

```diff
@@ -159,3 +160,37 @@ bool doesCollideWithBlob(CBlob@ this, CBlob@ blob)
 {
 	return blob.getShape().isStatic() && blob.isCollidable();
 }
+
+void onDie (CBlob@ blob) {
```

Comment:
You're pretty much repeating the exact same code as in the other file. Figure out a way to reuse most of this code.